pkg/minishift/addon/parser/command_handler.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package parser
+
+import (
+	"fmt"
+	"github.com/minishift/minishift/pkg/minishift/addon/command"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+const dockerCommand = "docker"
```

Comment:
started with a single const and as I added just duplicated the line. I will change the syntax